opba-banking-protocol-facade/src/main/java/de/adorsys/opba/protocol/facade/config/encryption/DatasafeConfig.java

```diff
@@ -102,6 +110,24 @@ public FintechConsentSpecSecureStorage fintechUserDatasafeServices(
         );
     }
 
+    @Bean
+    @ConfigurationProperties(prefix = "facade.datasafe")
+    public EncryptionConfig encryptionConfig() {
+        return EncryptionConfig.builder().build();
+    }
+
+    private EncryptionConfig buildEncryptionConfig() {
```

Comment:
Add log statement at INFO level to indicate from where (DB or YML) encryption configuration came

Reply:
Logging was added